Cleard blogpost

johnnewton-hub opened the pull request:
How did I do?

johnnewton-hub committed:
added Get Cleared image

I uploaded this file to the area I've been keeping my blog post images for consistency-Mark uses a different folder-hopefully this isn't an issue..

johnnewton-hub committed:
Create Cleard.md

Q+A session with week four co-winner Get Cleard

MandyMeindersma commented on _posts/Cleard.md:
```diff
@@ -0,0 +1,65 @@
+
+**Where are your team members from?**
+
+We are all from Canada. 
```
They were chatting with me one the phone last night, they are actually all from Calgary if we wanted to add that in

johnnewton-hub replied:
didn't want to stir up the rivalry, but sure

johnnewton-hub replied:
I think it's clear in her team description, and the link to the Cal Platform Accel.

johnnewton-hub committed:
Update Cleard.md

ratted them for being from Calgary

MandyMeindersma commented:
Let me know when you change the name and then look at what it really looks like, I assume there will be changes you want to make and then I'll peek after.

johnnewton-hub committed:
Rename Cleard.md to 2020-04-15-Cleard.md

johnnewton-hub committed:
Update 2020-04-15-Cleard.md

johnnewton-hub committed:
Update 2020-04-15-Cleard.md

ajyong commented on _posts/2020-04-15-Cleard.md:
```diff
@@ -0,0 +1,65 @@
+title: Week four co-winner Get Cleard, works to improve global screening processes
+---
+
+![Get Cleard](images/blog/Get Cleard.png)
```
This will have to be an absolute URL, you can verify that the image isn't found on our [deployment preview](digitaledmonton.slack.com).

It's easy: just add a forward slash at the start of the URL: `/images/blog/Get Cleard.png`

ajyong replied:
When you said "the image is too big", it confused me because the blog post itself should be resizing it.  I see that you were referring to the blog list page.  It's good practice to specify _where_ you were seeing the error.

<img width="1376" alt="image" src="https://user-images.githubusercontent.com/6334517/79385248-7f087680-7f25-11ea-8fd5-97d728aba32a.png">

I don't have a quick fix (okay I do but I have to finish what I started) and would suggest moving the image just above the first question.  This will cause the blog listing to use your first paragraph as an excerpt:

> This week's competition resulted in a tie...

johnnewton-hub replied:
got it...

johnnewton-hub replied:
I shouldn't have trusted the "copy path" button on Github clearly.  Tried to avoid miss-typing the path and got burned by copy/paste, and syntax ignorance of course  ; )

ajyong commented on _posts/2020-04-15-Cleard.md:
```diff
@@ -0,0 +1,65 @@
+
+![Get Cleard](images/blog/Get Cleard.png)
+
+This week’s competition resulted in a tie-a first for our Codevid-19 global hackathon. Jaime Wood kindly offered to answer a few questions about her team’s entry, [Get Cleard](https://www.cleard.ai/).
```
The dash here could either be an em-dash followed by a space: —

...or just your plain old colon and a space.  Right now it just looks like we've invented a new word called "tie-a" :)

johnnewton-hub replied:
Yes indeed it does look like some serious coinage as the grammarians say.  These weird little syntax things are stupid hard for me to catch right now.  That is about the last place I'd look for a prob., but of course...syntax.

ajyong commented on _posts/2020-04-15-Cleard.md:
```diff
@@ -0,0 +1,65 @@
+
+![Get Cleard](images/blog/Get Cleard.png)
+
+This week’s competition resulted in a tie-a first for our Codevid-19 global hackathon. Jaime Wood kindly offered to answer a few questions about her team’s entry, [Get Cleard](https://www.cleard.ai/).
```
Codevid-19 is stylized as CODEVID-19

ajyong commented on _posts/2020-04-15-Cleard.md:
```diff
@@ -0,0 +1,65 @@
+
+This week’s competition resulted in a tie-a first for our Codevid-19 global hackathon. Jaime Wood kindly offered to answer a few questions about her team’s entry, [Get Cleard](https://www.cleard.ai/).
+
+
```
Only add one blank line between each paragraph/heading.  When working locally, our default formatter will remove these extra blank lines.

With you working on GitHub.com, I suggest following this rule, submitting the PR, then reviewing your work on the [deploy preview](https://deploy-preview-90--codevid-19-staging.netlify.app/2020/04/15/cleard) version of the post.  If it doesn't look ideal, I'll be the bearer of bad news and say that what you were intending to make happen (i.e. more padding between sections) is out of your hands.  I could modify the HTML/CSS for you, but there's slightly higher priority work right now.

johnnewton-hub replied:
Fair enough.  I'll be moving local as absolutely quickly as possible-prob monday-i've got duelling courses this weekend (2 concurrent).  In the meantime, you bet I'll check in with netlify, a big improvement over github's preview btw.

johnnewton-hub commented:
Hi @ajyong , should have replied here-message is in that slack thread.  link above goes to website landing page, when I navigate to the blog listings page-my image is right at the top of my url to the article-not where it should be.  When I navigate to the article there's only an image placeholder.  I'll check in in a while...

ajyong commented on _posts/2020-04-15-Cleard.md:
```diff
@@ -0,0 +1,65 @@
+
+Frustration and fear of the lack of enhanced screening at airports. There needs to be a global pandemic outbreak plan. Clearly there was almost nothing in place for the coronavirus. The world needs to be prepared and be able to predict the next crisis. 
+
+
```
Remove extra blank

ajyong commented on _posts/2020-04-15-Cleard.md:
```diff
@@ -0,0 +1,65 @@
+  
+**How did your team come together?**
+
+
```
Remove extra blank

ajyong commented on _posts/2020-04-15-Cleard.md:
```diff
@@ -0,0 +1,65 @@
+
+**What tools, made available by the hackathon, are you using?**
+
+
```
Remove extra blank

ajyong commented on _posts/2020-04-15-Cleard.md:
```diff
@@ -0,0 +1,65 @@
+
+**Is there anything else you’d like to say?**
+
+
```
Remove extra blank (us devs are lazy and have Prettier auto-format our code so we can make mistakes like this and let the computer clean up for us)

ajyong commented on _posts/2020-04-15-Cleard.md:
```diff
@@ -0,0 +1,65 @@
+
+The outpouring of empathy and global cooperation is inspiring. Frontline workers from either end of the socioeconomic spectrum are risking their lives, and have become our leaders. We all hope the pain and suffering are not in vain, and that going forward this time of community, understanding, and respect continues; is not forgotten.  
+
+
```
Remove extra blank

ajyong replied:
This one got missed

ajyong commented on _posts/2020-04-15-Cleard.md:
```diff
@@ -0,0 +1,65 @@
+The outpouring of empathy and global cooperation is inspiring. Frontline workers from either end of the socioeconomic spectrum are risking their lives, and have become our leaders. We all hope the pain and suffering are not in vain, and that going forward this time of community, understanding, and respect continues; is not forgotten.  
+
+
+Website: [Get Cleard](https://www.cleard.ai/)<br>
```
Use of `<br>` here is fine.  Mark's suggestion of 2 blank lines actually causes some issues right now with local development, so I'd say keep it this way.

johnnewton-hub replied:
ahem...got lucky on that one!

johnnewton-hub committed:
Update 2020-04-15-Cleard.md

johnnewton-hub committed:
Update 2020-04-15-Cleard.md

em dash fixed

johnnewton-hub committed:
removed extra blank lines/moved image url

moved image url just above first question (to work with super secret hidden CSS template methinks)

johnnewton-hub committed:
Update 2020-04-15-Cleard.md

ajyong commented on _posts/2020-04-15-Cleard.md:
```diff
@@ -0,0 +1,60 @@
+
+This week’s competition resulted in a tie- a first for our Codevid-19 global hackathon. Jaime Wood kindly offered to answer a few questions about her team’s entry, [Get Cleard](https://www.cleard.ai/).
+
+![Get Cleard](/images/blog/Get Cleard.png)
```
Darn it, looks like we don't resize images properly yet.  I don't think I have the bandwidth to make the SASS edits required, so @johnny-newton can you replace this Markdown with HTML:

    <img src="/images/blog/Get Cleard.png" alt="Get Cleard" style="max-width: 100%;">

It'll fix this abomination:
<img width="1433" alt="image" src="https://user-images.githubusercontent.com/6334517/79393700-1fb16300-7f33-11ea-9ea4-d6c7a1f97edc.png">

ajyong reviewed (changes requested):
Missed one extra blank, and one change that is due to a lack of good stylesheeting 🤦

johnnewton-hub committed:
removed blank line

extra line was above "Website: etc." line- I think so, I'm having a bit of trouble identifying exactly which lines need removing from the comment.

johnnewton-hub committed:
Update 2020-04-15-Cleard.md

ajyong reviewed (approved):
LGTM, I have a bone to pick with our blog titles but that can be resolved later.

johnnewton-hub commented:
> LGTM, I have a bone to pick with our blog titles but that can be resolved later.

I think they are overly long-the last for sure..

johnnewton-hub commented:
does this mean we're at Squash and Merge?

ajyong commented:
I approved it, up to you to decide whether you'd like Mandy to also review and approve or not.

johnnewton-hub commented:
> I approved it, up to you to decide whether you'd like Mandy to also review and approve or not.

That makes sense as she was in early on this.  It's a "why ask me to review if you don't care about what I think in the end" potential misunderstanding.  Again etiquette, thanks.

ajyong committed:
Condense title, stylize CODEVID-19

ajyong commented:
[ch921]

johnnewton-hub commented:
HI @MandyMeindersma , I'd like to pull the trigger on this-it passed Aaron's thorough gauntlet, and I've got other fish to fry...
